LiteCore/Query/SQLiteQuery.cc

```diff
@@ -480,10 +499,12 @@ namespace litecore {
                         throw;
                 }
             }
+            LogTo(SQL, "bindParameters ends");
```

Comment:
What is the default log level of the SQL domain? 

@bmeike I don't remember if enabling verbose log from the platform will also enable SQL domain.

Reply:
I did not check. but it definitely work for when I set the callback level to Info